jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/testsuite/configuration/resources/DeleteExternalProperty.java

```diff
@@ -16,7 +16,11 @@
     public void delExternalProperties() {
         clickTab("Configuration");
         clickTab("Resources");
-        driver.findElement(By.xpath("//li/span[text()='ext.properties']/preceding-sibling::input[@type='checkbox']")).click();
+
+        // If not coming from add ext property, this code makes sure ext properties is the currently selected
+        // Topology node
+        driver.findElement(By.xpath("//span[text()='Ext Properties']")).click();
```

Comment:
Yes Arvindo. I only do a wait click when the previous action triggers the screen overlay with the gears.